.github/workflows/CI-workflow.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           flags: ml-commons
           token: ${{ secrets.CODECOV_TOKEN }}
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
I just merged this PR: https://github.com/opensearch-project/ml-commons/pull/3162